tests/lib/rules/indent.js

```diff
@@ -289,7 +291,20 @@ eslintTester.addRuleTest("lib/rules/indent", {
                 "        break;\n" +
                 "}",
             options: [4, {indentSwitchCase: true}],
-            errors: expectedErrors([[13, 4], [15, 4], [17, 4]])
+            errors: expectedErrors([[5, 8], [8, 8]])
+        },
+        {
+            code:
+                "switch (a) {\n" +
+                "case '1':\n" +
+                "b();\n" +
+                "break;\n" +
+                "default:\n" +
+                "c();\n" +
+                "break;\n" +
+                "}",
+            options: [4, {indentSwitchCase: true}],
+            errors: expectedErrors([[3, 4], [4, 4], [6, 4], [7, 4]])
```

Comment:
Please let me know whether you're sure that this should be part of the same rule/option. Initial issue reporter did expect all those to be errors, but after further discussion @nzakas said that he just expects breaks to be indented on the same level, so I'm not sure yet whether such a breaking change with cases is needed.